docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,388 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to Contribute to the Teleport Documentation
```

Comment:
Why is it "the Teleport documentation" in the title, but just "Teleport documentation" in the description? Is the additional "the" necessary in one place but not the other?

Reply:
Actually not needed in either place in my opinion.